src/api/article/article.model.js

```diff
@@ -5,22 +5,16 @@ import omit from 'lodash/omit';
 import { checkPermissions } from 'api/user';
 
 const ArticleSchema = new Schema({
-  title: {
-    type: String,
-    required: true,
-  },
-  subtitle: {
-    type: String,
-    required: true,
-  },
-  video: String,
-  image: String,
-  text: Schema.Types.Mixed,
   author: {
     type: Schema.Types.ObjectId,
     ref: 'User',
   },
-  tags: [String],
+  locales: [
```

Comment:
Array? not mapping `locale` -> `ArticleData`?

Reply:
I chose array to have `ref: 'ArticleData',` explicitly specified in the model.
Otherwise it's Schema.Types.Mixed and it's not clear at all if it's a map or whatever else.
Frontend should be fine with processing arrays: each `ArticleData` object has a `locale` field (yes, it makes it `O(N)`).